cmd/billing.go

```diff
@@ -322,6 +352,422 @@ func (c *config) clusterUsageCSV(cur *models.V1ClusterUsageRequest) error {
 	return nil
 }
 
+func (c *config) excel() error {
```

Comment:
should be a printer not a command

Reply:
We could rename this to "all", but I'm not sure how to generate an output for everything in the other formats.

Reply:
json and yaml output is simply a serialization, table output is obviously a bit harder, can then be return a error like `table output for all entities is not supported`

Reply:
maybe even better instead of `all` a slice of entities which should be returned, e.g. `cloudctl billing cluster,volume -o xlsx`